lib/matplotlib/widgets.py

```diff
@@ -295,12 +295,17 @@ def __init__(self, ax, label, valmin, valmax, valinit=0.5, valfmt=None,
         dragging : bool, default: True
             If True the slider can be dragged by the mouse.
 
-        valstep : float, default: None
-            If given, the slider will snap to multiples of *valstep*.
+        valstep : float or arraylike, default: None
```

Comment:
```suggestion
        valstep : float or array-like, default: None
```
(standard convention)